src/main/java/reposense/parser/AlphanumericArgumentType.java

```diff
@@ -0,0 +1,27 @@
+package reposense.parser;
+
+import java.util.regex.Pattern;
+
+import net.sourceforge.argparse4j.inf.Argument;
+import net.sourceforge.argparse4j.inf.ArgumentParser;
+import net.sourceforge.argparse4j.inf.ArgumentParserException;
+import net.sourceforge.argparse4j.inf.ArgumentType;
+
+/**
+ * Verifies a String is alphanumeric.
```

Comment:
A class usually _represents_ something rather than _does_ something. This class represents a Alphanumeric Argument Type